From: Benjamin Drung <benjamin.drung@ionos.com>
Date: Wed, 27 Oct 2021 13:08:33 +0200
Subject: Mark test_buildout_bootstrap with pytest

Pytest runs `BuildoutOnlineTestCase.setUpClass` despite skipping all
tests. Mark `test_buildout_bootstrap` with pytest to avoid running the
`setUpClass`.

Forwarded: https://github.com/saltstack/salt/pull/61133
Signed-off-by: Benjamin Drung <benjamin.drung@ionos.com>
---
 tests/unit/modules/test_zcbuildout.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/unit/modules/test_zcbuildout.py b/tests/unit/modules/test_zcbuildout.py
index 8845d53..b032bf3 100644
--- a/tests/unit/modules/test_zcbuildout.py
+++ b/tests/unit/modules/test_zcbuildout.py
@@ -401,7 +401,7 @@ class BuildoutOnlineTestCase(Base):
                 ]
             )
 
-    @skipIf(True, "TODO this test should probably be fixed")
+    @pytest.mark.skip(reason="TODO this test should probably be fixed")
     def test_buildout_bootstrap(self):
         b_dir = os.path.join(self.tdir, "b")
         bd_dir = os.path.join(self.tdir, "b", "bdistribute")
